File: ejercicios/ejercicio01/ejercicio01.java
package ejercicios;

public class ejercicio01 {

	public static int sumaElementos(int[] arreglo) {
        int suma = 0;
        for (int num : arreglo) {
            suma += num;
        }
        return suma;
    }

    public static void main(String[] args) {
        int[] arreglo = {1, 2, 3, 4, 5};
        int resultado = sumaElementos(arreglo);
        System.out.println("Suma: " + resultado);
    }

}

File: practica 01/ejercicios/ejercicio01/ejercicio01.cpp
#include <iostream>

int sumaElementos(int arreglo[], int tamaño) {
    int suma = 0;
    for (int i = 0; i < tamaño; ++i) {
        suma += arreglo[i];
    }
    return suma;
}

int main() {
    int arreglo[] = {1, 2, 3, 4, 5};
    int tamaño = sizeof(arreglo) / sizeof(arreglo[0]);
    int resultado = sumaElementos(arreglo, tamaño);
    std::cout << "Suma: " << resultado << std::endl;
    return 0;
}
